bq-storage-backend-init.tf

```diff
@@ -25,7 +25,7 @@ variable "billing_account_id" {
 
 locals {
   backend_folder_display_name = "${var.backend_prefix}-bq-driver-folder"
-  service_project_name = "${var.backend_prefix}-bq-driver"
+  service_project_name = "main-${var.backend_prefix}-bq-driver"
```

Comment:
upravil som este nazov main projektu pretoze sa mohlo stat ze to matchlo prefix stacku a pri mazani backedovych projektov v testoch to zmazlo aj main projekt